src/components/character-count/index.md

```diff
@@ -102,11 +102,9 @@ For example, ‘Summary must be 400 characters or less’.
 
 ## Research on this component
 
-The Government Digital Service (GDS) developed and tested this component in a prototype of the ‘Apply for a temporary event notice’ service.
+In 2017, the Government Digital Service (GDS) [developed and tested this component](https://github.com/alphagov/govuk-design-system/wiki/Character-count-testing-and-user-research) in a prototype of the ‘Apply for a temporary event notice’ service. During this research, the component was tested with 17 users, including those with low digital skills and users with disabilities.
 
-During this research, the component was tested with 17 users, including those with low digital skills and users with disabilities.
-
-[Read more about this research in the Design System wiki on GitHub](https://github.com/alphagov/govuk-design-system/wiki/Character-count-testing-and-user-research).
+In 2022, the GOV.UK Design System team [updated the component to make it more accessible](https://dav-idc.com/making-a-character-count-component-more-accessible/), stopping the character count being announced twice by some screenreaders.
```

Comment:
```suggestion
In 2022, the GOV.UK Design System team updated the component to make it more accessible, stopping the character count being announced twice by some screenreaders.
```

Traditionally we don’t link out to personal blogs, only official ones. This is in line with the [GOV.‌UK external link policy](https://www.gov.uk/guidance/content-design/links).

Reply:
Interesting, did not know this. I had in mind the links in the [names component](https://design-system.service.gov.uk/patterns/names/) and the link to the video for the Select one.

Is there an official write-up of this? Could link to Pull Request or the Issue but that’s a bit harder to follow.

Reply:
Nothing on the official blogs platform, no. 

The blog post is great (obviously!) but it’s the story of a piece of work that happened on the component. The purpose of our guidance is to help teams understand when and how to use the components. The links on the names component teach teams about common problems with names, but I don’t think David’s blog post fulfils a similar need. 

Let me know if you think that’s an incorrect assessment.

Reply:
@stevenjmesser Up to you and the team, I just thought it was useful to mention the additional research done into screenreaders using the component, and the updates that were subsequently done. Will help any teams with any queries about the component during accessibility audits!

The blog post adds a ton of this context. If it can’t be linked to, maybe link to the pull request or issue instead? Or could we ask David and beeps if it could be copied to a GitHub wiki page like the other, older link?

Reply:
I'm happy with whatever, this is the reason it was published externally in the first place though. We weren't really able to identify a solid audience and message in order to publish on a UK Gov blog. And agreed that the design system website doesn't make a habit of linking to informal sources.

Reply:
@frankieroberto would the comment 'update' we made on the character count backlog item cover enough of the info you're thinking should be referenced in the design system guidance page?

https://github.com/alphagov/govuk-design-system-backlog/issues/67#issuecomment-1131719459

Reply:
@frankieroberto I’m not on the team any more but you’ll need to commit the suggestion (or not) and resolve this conversation for the team to merge it.

Reply:
I'll update the link to point to the [GitHub issue comment](https://github.com/alphagov/govuk-design-system-backlog/issues/67#issuecomment-1131719459) instead. I kinda think that’s less user-friendly than a blog post, but I guess it might be more likely to stick around long-term? (Although I'm sure @dav-idc will keep his blog going! 😄).